program.py

```diff
@@ -0,0 +1,11 @@
+ #GCD of 2 numbers 
+ 
+ import math 
+  
```

Comment:
your math is not indented and there is conflict with the file. Please make the file to gcd_ayush.py